### DISCUSSION TITLE
clearing results to be able to parse multiple xml files?

### DISCUSSION THREAD

**Discussion by ownlyme** [manoelcampos/xml2lua#96] [Q&A] [answered]
when i parse multiple xml files, the additional files seem to get added to the same table and there doesn't seem to be a way to clear the results

**Comment by tuananh** [comment]
i have the same issue. how do i clear it before i can pass another xml string?

**Comment by nilzao** [comment, marked as answer]
the trick is 
```lua
handler = handler:new()
```

edit example3.lua, add these lines at end to see the example:

```lua
print("\nUsing New handler instance")
handler = handler:new()
parser = xml2lua.parser(handler)
parser:parse(xml)
xml2lua.printable(handler.root)
```
